src/main/java/Cubone.java

```diff
@@ -0,0 +1,89 @@
+                // mark task as undone
+                String[] split = input.split(" ");
+                int index = Integer.parseInt(split[1]);
+                inputed_tasks.get(index-1).markAsUndone();
+                System.out.println(chat_bar + chat_prefix + "OK, I've marked this task as not done yet:\n" + inputed_tasks.get(index-1).toString() + "\n" + chat_bar);
+            } else if (input.contains("todo")){
+                // add todo task
+                String[] split = input.split(" ");
+                if(split.length == 1){
+                    System.out.println(chat_bar + chat_prefix + "☹ OOPS!!! The description of a todo cannot be empty.\n" + chat_bar);
+                }else{
+                    inputed_tasks.add(new Todo(input));
+                    System.out.println(chat_bar + chat_prefix + "Got it. I've added this Todo:\n" + inputed_tasks.get(inputed_tasks.size()-1).toString() + "\n" +
+                                        "now you have " + inputed_tasks.size() + " tasks in the list\n" + chat_bar); 
+                }
+            } else if (input.contains("deadline")){
+                // add deadline task
+                // usage: deadline <description> /by <date>
+                Pattern deadlinePattern = Pattern.compile("deadline (.+?) /by (.+)");
+                Matcher deadlineMatcher = deadlinePattern.matcher(input);
+                if (deadlineMatcher.find()) {
+                    inputed_tasks.add(new Deadline(deadlineMatcher.group(1), deadlineMatcher.group(2)));
+                    System.out.println(chat_bar + chat_prefix + "Got it. I've added this task:\n" + inputed_tasks.get(inputed_tasks.size()-1).toString() + "\n" + 
+                                        "now you have " + inputed_tasks.size() + " tasks in the list\n" + chat_bar);
+                } else {
+                    System.out.println(chat_bar + chat_prefix + "☹ OOPS!!! Someting missing, usage: deadline <description> /by <date>\n" + chat_bar);
+                }
+            } else if (input.contains("event")){
+                // add event task
+                // usage: event <description> /from <date> /to <date>
+                Pattern eventPattern = Pattern.compile("event (.+?) /from (.+?) /to (.+)");
+                Matcher eventMatcher = eventPattern.matcher(input);
+                if (eventMatcher.find()) {
+                    inputed_tasks.add(new Event(eventMatcher.group(1), eventMatcher.group(2), eventMatcher.group(3)));
+                    System.out.println(chat_bar + chat_prefix + "Got it. I've added this task:\n" + inputed_tasks.get(inputed_tasks.size()-1).toString() + "\n" + 
+                                        "now you have " + inputed_tasks.size() + " tasks in the list\n" + chat_bar);
+                } else {
+                    System.out.println(chat_bar + chat_prefix + "☹ OOPS!!! Someting missing, usage: event <description> /from <date> /to <date>\n" + chat_bar);
```

Comment:
There is some nesting within the conditional blocks. The nested `if` statements could be flattened by refactoring each conditional block into its own method. This would improve the readability and reduce the complexity of the code.